commons-io/pom.xml

```diff
@@ -140,7 +140,7 @@
 										<version>${core.artifact.version}</version>
 									</artifact>
 									<moduleInfoSource>
-										open module org.apache.commons.io {
+										open module dev.ikm.jpms.commons.io {
```

Comment:
Shouldn't the module name remain `org.apache.commons.io`? Generally, only the artifact `groupId` is changed to `dev.ikm.jpms` to reference the JPMS compliant version.

Reply:
so you will encounter module runtime collisions if you do this, when in theory, they could be both loaded (though it is not suggested either).  In general, much like maven artifact groups, best practice is to start it with your domain if you are the owner.  Similarly, we will not and cannot publish to maven central under `org.apache` because we do not own this domain.

Reply:
Thank you for the clarification, @jlgrock. We now understand why you’ve chosen to use your own domain for both the groupId and the module name when repackaging or distributing libraries.